src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -144,7 +144,6 @@ Public Class BuildDevDivInsertionFiles
         "Microsoft.VisualStudio.LanguageServices.dll",
         "Microsoft.VisualStudio.LanguageServices.Implementation.dll",
         "Microsoft.VisualStudio.LanguageServices.VisualBasic.dll",
-        "Microsoft.VisualStudio.Platform.VSEditor.Interop.dll",
```

Comment:
> "Microsoft.VisualStudio.Platform.VSEditor.Interop.dll", [](start = 7, length = 56)

not related to this change directly, but required because in the latest 15.6 editor bits this dll is gone (it's now an embeddable design time dll).